libs/dh/admin/feature-edit-user-role-modal/src/lib/dh-edit-user-role-modal.component.html

```diff
@@ -25,9 +25,14 @@
       <watt-tabs>
         <watt-tab [label]="t('tab.masterData.tabLabel')">
           <div class="form-wrapper">
-            <watt-form-field>
+            <watt-form-field class="watt-space-stack-l">
               <watt-label>{{ t("tab.masterData.nameInputLabel") }}</watt-label>
               <input wattInput type="text" formControlName="name" />
+              <watt-error
+                *ngIf="userRoleEditForm.get('name')?.errors?.['nameAlreadyExists']"
```

Comment:
Suggestion: Can't you use "userRoleEditForm.controls.name.errors.['nameAlreadyExists']" it avoids atleast one magic string, and gives a compilation error should the name of the formcontrol change.

Reply:
Good point. Done ✅